app/scripts/controllers/preferences.js

```diff
@@ -449,7 +449,8 @@ export default class PreferencesController {
    */
   syncAddresses(addresses) {
     if (!Array.isArray(addresses) || addresses.length === 0) {
-      throw new Error('Expected non-empty array of addresses. Error #11201');
+      // eslint-disable-next-line @metamask/design-tokens/color-no-hex
+      throw new Error('Expected non-empty array of addresses. Error #11201'); // not a hex color value
```

Comment:
There is a current issue with some false positives that need to be fixed issue here https://github.com/MetaMask/eslint-plugin-design-tokens/issues/20